content/finalist/ttjc@kaushalkoladiya/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Kaushal Koladiya
+type: "finalist"
+name: "Kaushal Koladiya"
+portfolio: "https://kaushalkoladiya.netlify.app/"
+github-link: "https://github.com/kaushalkoladiya"
+linkedin-link: "https://www.linkedin.com/in/kaushal-koladiya-a65676179/"
+looking-for: "job"
+description: "React, Node"
+---
+
+Hii! I am Kaushal Koladiya, I'm final year BCA student and self learner.
+
+### introduction
+
+I graduated from High School for the Gifted and got into a Veer Narmad South Gujarat University because I think 3-year study at this university would prepare me well.
+
+## why are you taking this challenge?
+
+I am looking for job.In other words I'm hungry for new knowledge and technologies.
+
+## something interesting about you?
+
+Like to watch technology video.
+Listening music like pop, rock, etc.
+Good catch up power to anything new.
+
+### help I need?
+
+I have little bit knowledge about API but I wanna learn more about it.
+
+### how to contact me?
+
+- [Twitter](https://twitter.com/kaushal__p)
+- [LinkedIn](https://www.linkedin.com/in/kaushal-koladiya-a65676179/)
+
+### projects
+
+#### News App with React
+
+_description_ It is a normal news app by using API.
+
+_stack_ React
+
+_hosted link_ https://newsglobe.netlify.app
+
+_github link_ https://github.com/kaushalkoladiya/newsglobe
```

Comment:
there's some issue with this application have a look

Reply:
It's fixed now, please check it

Reply:
> there's some issue with this application have a look

It ready you can check it.